js/shared.js

```diff
@@ -147,6 +147,7 @@ const signInFlowRender = (signInEmail) => {
 };
 
 export const sendEmailLink = () => {
+    const preferredLanguage = window.localStorage.getItem("preferredLanguage");
```

Comment:
please use the function that has already been created to get preferred language

Reply:
Solved